pkg/istio/client.go

```diff
@@ -27,23 +26,18 @@ const (
 )
 
 var (
-	vsHost                     = "listener.kyma.cloud.sap" //nolint:gochecknoglobals
-	errNoGatewayConfigured     = errors.New("error processing Watcher: No istio gateway configured")
-	errCantFindMatchingGateway = errors.New("can't find matching Istio Gateway")
+	errNoGatewayConfigured        = errors.New("error processing Watcher: No istio gateway configured")
```

Comment:
to make the errors assertable, I think they should always be exported.

Reply:
done